home-work-app/webpack.config.dev.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const webpack = require('webpack');
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CaseSensitivePathPlugins = require('case-sensitive-paths-webpack-plugin');
+
+module.exports = {
+    mode: 'development',
```

Comment:
Let's add `devServer: { open: true }` to automatically launch the browser on application start

Reply:
done